src/NuGet.Core/NuGet.Commands/RestoreCommand/LockFileBuilder.cs

```diff
@@ -614,7 +614,7 @@ private static void PopulatePackageFolders(IEnumerable<string> packageFolders, L
             lockFile.PackageFolders.AddRange(packageFolders.Select(path => new LockFileItem(path)));
         }
 
-        private static LockFileLibrary CreateLockFileLibrary(LocalPackageInfo package, string sha512, string path)
+        internal static LockFileLibrary CreateLockFileLibrary(LocalPackageInfo package, string sha512, string path)
```

Comment:
I've used this method in a lot of my profiling. I've found this extremely useful, since I can have tests and other assemblies (such as benchmarks call this method).